cmsplugin_profile/static/cmsplugin_profile/sass/profilegrid.scss

```diff
@@ -936,12 +943,46 @@ $grid: ".og-grid";
             }
         }
     }
-    @media (min-width: 768px) and (max-width: 959px) {
+    @media (width: 1024px) {
         body {
             &.studio, &:not(.studio) {
-                .span8.page-content, .span12.page-content {
+                .span8.page-content {
                     .pbs-profile-promo {
-                        .og-grid {
+                        .og-grid:not(.og-4-items) {
+                            li, li > a {
+                                width: 309px;
+                                height: 309px;
+                                line-height: 307px;
+                            }
+                            li:nth-child(3n) {
+                                margin-right: 0;
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+    body {
+        &.studio, &:not(.studio) {
```

Comment:
Still seems wrong to me. Maybe you should use `!important` that way it's clear you want to override something